crates/server/src/sync.rs

```diff
@@ -51,11 +362,22 @@ pub(crate) fn sync_from_claude(
         .map(|p| p.join("agents"))
         .unwrap_or_else(|| mirror_root.join("../agents"));
     // Track directories we've already mirrored to avoid repeated work when multiple SKILLs exist.
-    let mut mirrored_dirs: std::collections::HashSet<PathBuf> = std::collections::HashSet::new();
+    let mut mirrored_dirs: HashSet<PathBuf> = HashSet::new();
     for entry in WalkDir::new(claude_root)
         .min_depth(1)
         .max_depth(8)
```

Comment:
```suggestion
        .max_depth(20)
```

to be consistent with the max depth above

Reply:
✅ Fixed - changed max_depth from 8 to 20 for consistency